includes/features/stanford_jumpstart_home/homepage-palm-dynamic.feature

```diff
@@ -0,0 +1,43 @@
+Feature: Palm Homepage Layout
+  In order to ensure that the Jumpstart "Palm" dynamic page layout is configured correctly
+  As an end user
+  I want to check for the existence of content that should appear
+
+@api
```

Comment:
Probably should add a @dev and @destructive to this as it does configuration change.